staging/src/k8s.io/kubectl/pkg/cmd/config/create_cluster.go

```diff
@@ -181,9 +187,6 @@ func (o createClusterOptions) validate() error {
 		if caPath == "" {
 			return fmt.Errorf("you must specify a --%s to embed", clientcmd.FlagCAFile)
 		}
-		if _, err := ioutil.ReadFile(caPath); err != nil {
```

Comment:
can we os.Stat here to validate it exists?

Reply:
Yes, that appears to work fine, even with a fifo... I'll make that change

Reply:
Good idea!